network/topics/controller_test.go

```diff
@@ -47,7 +49,7 @@ func TestTopicManager(t *testing.T) {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 
-		validator := validation.NewMessageValidator(networkconfig.TestNetwork)
+		validator := &DummyMessageValidator{}
```

Comment:
What is the issue with using `validation.NewMessageValidator(networkconfig.TestNetwork)`?

Reply:
@nkryuchkov , the message validation flow tests for consensus logic and operator signatures, these don't exist in this test and not really needed. until now the TestNetwork was without the permissionless fork aka without sig verification.

Reply:
Enabling strict message validation requires complete rework of the test. This should be done in another PR